build/specifications/Docker.json

```diff
@@ -8270,6 +8271,238 @@
+          },
+          {
+            "name": "MemorySwap",
+            "type": "long?",
+            "format": "--memory-swap {value}",
+            "help": "Swap limit equal to memory plus swap: '-1' to enable unlimited swap."
+          },
+          {
+            "name": "Network",
+            "type": "string",
+            "format": "--network {value}",
+            "help": "Set the networking mode for the RUN instructions during build."
+          },
+          {
+            "name": "NoCache",
+            "type": "bool",
+            "format": "--no-cache",
+            "help": "Do not use cache when building the image."
+          },
+          {
+            "name": "Output",
+            "type": "string",
+            "format": "--output {value}",
+            "help": "Output destination (format: type=local,dest=path)."
+          },
+          {
+            "name": "Platform",
+            "type": "string",
+            "format": "--platform {value}",
+            "help": "Set platform if server is multi-platform capable."
+          },
+          {
+            "name": "Progress",
+            "type": "string",
+            "format": "--progress {value}",
+            "help": "Set type of progress output (auto, plain, tty). Use plain to show container output."
+          },
+          {
+            "name": "Pull",
+            "type": "bool",
+            "format": "--pull",
+            "help": "Always attempt to pull a newer version of the image."
+          },
+          {
+            "name": "Push",
+            "type": "bool",
+            "format": "--push",
+            "help": "Shorthand for --output=type=registry"
+          },
+          {
+            "name": "Quiet",
+            "type": "bool",
+            "format": "--quiet",
+            "help": "Suppress the build output and print image ID on success."
+          },
+          {
+            "name": "Rm",
+            "type": "bool",
+            "format": "--rm",
+            "help": "Remove intermediate containers after a successful build."
+          },
+          {
+            "name": "Secret",
+            "type": "string",
+            "format": "--secret {value}",
+            "help": "Secret file to expose to the build (only if BuildKit enabled): id=mysecret,src=/local/secret."
+          },
+          {
+            "name": "SecurityOpt",
+            "type": "List<string>",
+            "format": "--security-opt {value}",
+            "help": "Security options."
+          },
+          {
+            "name": "ShmSize",
+            "type": "long?",
+            "format": "--shm-size {value}",
+            "help": "Size of /dev/shm."
+          },
+          {
+            "name": "Ssh",
+            "type": "string",
+            "format": "--ssh {value}",
+            "help": "SSH agent socket or keys to expose to the build (only if BuildKit enabled) (format: default|&lt;id&gt;[=&lt;socket&gt;|&lt;key&gt;[,&lt;key&gt;]])."
```

Comment:
this can be its own type `DockerSshMethod` or something similar. Check for `enumerations` in other specification files.

Reply:
There is actually only one predefined value `default` other values are custom key values 
```
docker build --ssh projecta=./projecta.pem --ssh projectb=./projectb.pem
```

Reply:
okay got it